app/api/odm/MultiTenantMongooseModel.ts

```diff
@@ -38,20 +45,20 @@ class MultiTenantMongooseModel<T> {
 
   async findOneAndUpdate(
     query: UwaziFilterQuery<T>,
-    update: Readonly<Partial<T>> & { _id?: any },
-    options: any = {}
+    update: UwaziUpdateQuery<T>,
+    options: UwaziQueryOptions
   ) {
     return this.dbForCurrentTenant().findOneAndUpdate(query, update, options);
   }
 
-  async create(data: Readonly<Partial<T>> & { _id?: any }) {
+  async create(data: PartialDataType<T>) {
```

Comment:
Nothing in the new implementation has `Readonly`.  Any hint as to why this could have been needed?  I just removed it from everywhere and nothing is failing.  I'm open to being enlightened about this as well.

Reply:
not sure, @fnocetti @mfacar ?

Reply:
I don't think it's needed for the permissions model 🤔